.github/workflows/push.yml

```diff
@@ -13,12 +13,24 @@ on:
     # seed the build cache.
     branches:
       - main
+  schedule:
+    - cron: '0 0,12 * * *'  # Runs at 00:00 and 12:00 UTC daily
 
 env:
   GOTESTSUM_FORMAT: github-actions
 
 jobs:
+  cleanups:
+    runs-on: ubuntu-latest
+    steps:
+      - name: Clean up cache if running on schedule
+        if: ${{ github.event_name == 'schedule' }}
```

Comment:
You can move this to the job level.

Reply:
IT does not work, it skipped all the dependent jobs.